flex/utils/service_utils.cc

```diff
@@ -17,58 +17,6 @@
 
 namespace gs {
 
```

Comment:
Those method are inlined to avoid linking flex_utils when doing codegen for cypher queries.